pipeline/transformer/transformer_common/src/main/scala/weco/catalogue/transformer/TransformerWorker.scala

```diff
@@ -119,22 +117,58 @@ trait TransformerWorker[Payload <: SourcePayload, SourceData, SenderDest]
       // nightly Sierra reharvest, when ~250k Sierra source records get synced with
       // Calm.  The records get a new modifiedDate from Sierra, but none of the data
       // we care about for the pipeline is changed.
-      case Right((newWork, key)) =>
+      case Right((transformedWork, key)) =>
         retriever
-          .apply(workIndexable.id(newWork))
+          .apply(workIndexable.id(transformedWork))
           .map { storedWork =>
-            if (newWork.shouldReplace(storedWork)) {
-              Right(Some((newWork, key)))
+            val shouldSend = {
```

Comment:
nitpicking but I would prefer this in a separate `shouldSend` method so that `compareToStored` is easier to read

Reply:
Done.